src/Avalonia.Controls/TransitioningContentControl.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Threading;
+using Avalonia.Animation;
+using Avalonia.Controls.Templates;
+using Avalonia.Threading;
+
+namespace Avalonia.Controls;
+
+/// <summary>
+/// Displays <see cref="ContentControl.Content"/> according to a <see cref="FuncDataTemplate"/>.
+/// Uses <see cref="PageTransition"/> to move between the old and new content values. 
+/// </summary>
+public class TransitioningContentControl : ContentControl
```

Comment:
I don't think it can be merged before same control in ReactiveUI repo will be merged. They are also in the same xaml namespace.

Reply:
reactive one needs an xmlns namespace to use it.